jwst/photom/photom.py

```diff
@@ -347,14 +407,14 @@ def calc_miri(self, ftab):
                                                 dqflags.pixel['NON_SCIENCE'])
 
             # Divide the science data and err by the conversion factors
-            self.input.data /= ftab.data*ftab.pixsiz
-            self.input.err /= ftab.data*ftab.pixsiz
+            self.input.data /= ftab.data * ftab.pixsiz
+            self.input.err /= ftab.data * ftab.pixsiz
```

Comment:
Just a suggestion, but since you compute ftab.data * ftab.pixsiz a few lines below, maybe you could move that line farther up and then divide data and err by self.input.relsens2d, instead of duplicating the arithmetic.